common/persistence/sql/sqlplugin/sqlite/task.go

```diff
@@ -341,8 +345,56 @@ func (mdb *db) UpdateTaskQueueUserData(ctx context.Context, request *sqlplugin.U
 	return nil
 }
 
+func (mdb *db) AddBuildIdToTaskQueueMapping(ctx context.Context, request sqlplugin.AddBuildIdToTaskQueueMapping) error {
+	query := addBuildIdToTaskQueueMappingQry
+	var params []any
+	for idx, buildId := range request.BuildIds {
+		if idx == len(request.BuildIds)-1 {
+			query += "(?, ?, ?)"
+		} else {
+			query += "(?, ?, ?), "
+		}
+		params = append(params, request.NamespaceID, buildId, request.TaskQueueName)
+	}
+
+	_, err := mdb.conn.ExecContext(ctx, query, params...)
+	return err
+}
+
+func (mdb *db) RemoveBuildIdToTaskQueueMapping(ctx context.Context, request sqlplugin.RemoveBuildIdToTaskQueueMapping) error {
+	query := removeBuildIdToTaskQueueMappingQry
+	var params []any
+	for idx, buildId := range request.BuildIds {
+		query += "namespace_id = ? AND build_id = ? AND task_queue_name = ?"
+		if idx < len(request.BuildIds)-1 {
+			query += " OR "
+		}
+		params = append(params, request.NamespaceID, buildId, request.TaskQueueName)
+	}
+
+	_, err := mdb.conn.ExecContext(ctx, query, params...)
+	if err == nil {
+		// TODO(bergundy)
+		panic("this should be properly tested once we support deletion")
```

Comment:
Intentional, I started implementing it but realized I can't test it properly and decided to leave this panic as a reminder that this has to be tested.

Reply:
Sure, I didn't think it'd matter much because it's a placeholder for when we do implement deletions.